public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -19,29 +23,93 @@ export class StoryQuestionsQuestion extends React.Component {
     onFormErrorsUpdate: PropTypes.func
   };
 
+  constructor(props) {
+    super(props);
```

Comment:
Constructor blocks aren't required because of the `transform-class-properties` babel plugin. Props are already applied as `this.props` and state (or anything for that matter) can be set as a called properly by simply using:
```
state = {
...
}
```

inside the class declaration.